samples/cpp/continuous_batching_multiple_generation_sample/continuous_batching_multiple_generation_sample.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <openvino/openvino.hpp>
+#include <cxxopts.hpp>
+
+#include "openvino/genai/continuous_batching_pipeline.hpp"
+
+void print_generation_result(const ov::genai::GenerationResult& generation_result) {
+    for (size_t output_id = 0; output_id < generation_result.m_generation_ids.size(); ++output_id) {
+        std::cout << "Answer " << output_id << " (" << generation_result.m_scores[output_id] << ") : " << generation_result.m_generation_ids[output_id] << std::endl;
+    }
+}
+
+int main(int argc, char* argv[]) try {
+    // Command line options
+
+    cxxopts::Options options("accuracy_sample", "Help command");
+
+    options.add_options()
+    ("n,num_prompts", "A number of prompts", cxxopts::value<size_t>()->default_value("1"))
+    ("dynamic_split_fuse", "Whether to use dynamic split-fuse or vLLM scheduling", cxxopts::value<bool>()->default_value("false"))
+    ("m,model", "Path to model and tokenizers base directory", cxxopts::value<std::string>()->default_value("."))
+    ("h,help", "Print usage");
+
+    cxxopts::ParseResult result;
+    try {
+        result = options.parse(argc, argv);
+    } catch (const cxxopts::exceptions::exception& e) {
+        std::cout << e.what() << "\n\n";
+        std::cout << options.help() << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    if (result.count("help")) {
+        std::cout << options.help() << std::endl;
+        return EXIT_SUCCESS;
+    }
+
+    const size_t num_prompts = 1;
+    const bool dynamic_split_fuse = false;
+    const std::string models_path = "/home/panas/llm/models/opt-125m/";
+
+    // create dataset
+
+    std::vector<std::string> prompt_examples = {
+        "What is OpenVINO?\n",
+        "Explain this in more details?\n",
+        "What is your name?\n",
+        "Tell me something about Canada\n",
+        "What is OpenVINO?\n",
```

Comment:
It seems that the only thing this sample features is prompts starting with the same prefix. Sample isn't need for that. You should just describe the applied optimization in other sample readme and code comments.

Reply:
Removed the sample. 
Regarding description, I suppose we need to add it to CB chat readme when this PR is merged: https://github.com/openvinotoolkit/openvino.genai/pull/644